RELEASENOTES/v2.0.0.rc-1.MD

```diff
@@ -0,0 +1,64 @@
+# Notary v2 Release Candidate 1 
+
+## Overview
+Notary v2 working group is pleased to announce its first release candidate, RC-1. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
```

Comment:
The tag will be `https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0-rc.1`